__tests__/app.test.js

```diff
@@ -405,4 +405,39 @@ describe("GET /api/users", () => {
           expect(body.msg).toBe("Resource Not Found");
         });
     });
+  });
+
+  describe('GET /api/articles?topic', () => {
+    test('200 - responds with an array of article objects filtered by the topic query', () => {
+      return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).not.toHaveLength(0);
+          articles.forEach((article) => {
+            expect(article).toHaveProperty('topic', 'mitch');
+          });
+        });
+    });
+  
+    test('200 - responds with all articles when topic query is not provided', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).not.toHaveLength(0);
+        });
+    });
```

Comment:
This should already be tested for in a previous test when you first created the /api/articles endpoint so it does not need to be tested again here.